src/pager.js

```diff
@@ -13,6 +13,7 @@ export class Pager {
   @bindable criteria  = {}; // search criteria for DB resource
   @bindable resource;       // data resource, either a ORM or a array
   @bindable pages;          // total amount of pages
+  @bindable resultCount = 0;
```

Comment:
Rename to resourceCount and move up to the other bindables with defaults (like criteria). Also add small comment.